### DISCUSSION TITLE
Any plan to revoke ssl certificate effected by CAA rechecking bug?

### DISCUSSION THREAD

**Discussion by hanabi1224** [vercel/vercel#3902] [General] [answered]
I saw cert of my web app was issued on 2/5/2020 which seems to be effected by [CAA Rechecking Bug](https://community.letsencrypt.org/t/2020-02-29-caa-rechecking-bug/114591).   Thanks!
![image](https://user-images.githubusercontent.com/6546635/76424952-e5a8ea80-63e3-11ea-947c-17a04de8f6b8.png)

**Comment by paulogdm** [comment, marked as answer]
Hello,

All certificates affected by any upstream bug were promptly removed. Let's Encrypt also revoked a few certificates which didn't affect any of our deployments.

Can you elaborate a little bit?

**Reply by hanabi1224** [reply]
Good to know it's solved. Thanks!
